src/act.obj.cpp

```diff
@@ -3511,7 +3511,7 @@ ACMD(do_activate)
 
       int delta = ((int)(GET_REAL_MAG(ch) / 100) * 100) - GET_POWER_POINTS(ch);
       if (total > delta)
-        send_to_char(ch, "That costs %d points to activate, but you only have %d free.\r\n", (total / 100), (delta / 100));
```

Comment:
These should be cast as (float) as well, with %.2f instead of %.2d (otherwise it'll perpetually display things like 2.00 for 2.50, etc)